centreon/tests/e2e/features/Cloud-notifications/01-notification-create.feature

```diff
@@ -0,0 +1,52 @@
+@ignore
+@REQ_MON-16477
+Feature: Creating a Notification Rule
+  As a Centreon user with access to the Notification Rules page
+  The user needs to create a Notification Rule
+  So that the configured users are notified of status changes on configured resources
+
+  Background:
+    Given a user with access to the Notification Rules page
+    And the user is on the Notification Rules page
+
+  Scenario: Creating a host group Notification Rule
+    Given a host group with hosts
+    When the user defines a name for the rule
+    And the user selects a host group and hosts with associated events on which to notify
+    And the user selects some contacts and/or contact groups
```

Comment:
missing precision : `and / or`
when the dev will implement the test, he won't know which scenario to choose
need to choose between `and` or `or`, or maybe add scenario outline